docs/pages/resources/spells.de.mdx

```diff
@@ -0,0 +1,89 @@
+# Zaubersprüche
+
+Diese Ressource bietet Informationen über Zaubersprüche aus dem Harry-Potter-Universum.
+
+## Attribute
+
+{
+<table>
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>Typ</th>
+      <th>Beschreibung</th>
+      <th>Filterbar</th>
+    </tr>
+  </thead>
+  <tbody>
+    {[
+      {
+        name: "category",
+        type: "Zeichenfolge",
+        description: "Kategorie des Zauberspruchs.",
+        filterable: true,
+      },
+      {
+        name: "creator",
+        type: "Zeichenfolge",
+        description: "Schöpfer/in des Zauberspruchs.",
+        filterable: true,
+      },
+      {
+        name: "effect",
+        type: "Zeichenfolge",
+        description: "Effekt des Zauberspruchs.",
+        filterable: true,
+      },
+      {
+        name: "hand",
+        type: "Zeichenfolge",
+        description: "Handbewegung des Zauberspruchs.",
+        filterable: true,
+      },
+      {
+        name: "image",
+        type: "Zeichenfolge",
+        description: "Link zu einem Bild des Zauberspruchs.",
+        filterable: false,
+      },
+      {
+        name: "incantation",
+        type: "Zeichenfolge",
+        description: "Beschwörungsformel für den Zauberspruch.",
+        filterable: true,
+      },
+      {
+        name: "light",
+        type: "Zeichenfolge",
+        description: "Licht des Zauberspruchs.",
+        filterable: true,
+      },
+      {
+        name: "name",
+        type: "Zeichenfolge",
+        description: "Name des Zauberspruchs.",
+        filterable: true,
+      },
+      {
+        name: "slug",
+        type: "Zeichenfolge",
+        description: Slug, um den Zauberspruch zu identifizieren.",
```

Comment:
@kolja-lucht the issue with the build probably lies here.

Reply:
Thanks, I already pushed the change. The quote in front of slug was missing. Build is fixed